cluster/test-deploy/README.md

```diff
@@ -10,24 +10,26 @@ should have:
 
 See `data/vars.yaml` for the default settings.  This depends on the `openshift/origin-gce:latest` image
 which is a pre-built version of the GCE reference architecture and the openshift-ansible playbooks for the
-appropriate release.
+appropriate release. That image is built from https://github.com/openshift/origin-gce.
 
-```
-$ cd data
+Other prerequisites:
+
+* Docker 1.12+ installed and available
```

Comment:
Do we want to say 1.12.6 for good measure

Reply:
I don't think it matters in this particular case - 1.12 is so old already most people would be like "wth"